source/types/hooks.ts

```diff
@@ -8,7 +8,6 @@ export type BeforeRequestHook = (
 
 export type BeforeRetryState = {
 	request: Request;
-	response: Response;
```

Comment:
I'm not sure if response is needed

In docs it's not mentioned, in tests I didn't find a usage too